packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -11,6 +11,9 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 - Changed the hashing algorithm for e-tag generation from `sha1` to `sha256` to avoid collisions. [#2890](https://github.com/zowe/zowe-explorer-vscode/pull/2890)
 - Updated the FTP plugin dependency to `3.0.0-next.202403191358` for technical currency [#2783](https://github.com/zowe/vscode-extension-for-zowe/pull/2783).
 - Update Zowe SDKs to `8.0.0-next.202405241828` for technical currency. [#2918](https://github.com/zowe/zowe-explorer-vscode/issues/2918)
+- Renamed extension to `IBM z/OS FTP Plug-in for Zowe`. [#2990](https://github.com/zowe/zowe-explorer-vscode/issues/2990)
```

Comment:
Should this say `IBM z/OS FTP for Zowe Explorer` to match the new display name? 😋